databases/week2/Part3.sql

```diff
@@ -0,0 +1,25 @@
+USE hyf_lesson2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.title, user.email FROM task JOIN user_task ON task.id = user_task.task_id JOIN user ON user_task.user_id = user.id
+WHERE email LIKE '%@spotify.com';
+
+--Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT  task.id ,task.title,status.name AS status_name, user.name AS user_name
+FROM task 
+JOIN status ON task.status_id = status.id 
+JOIN user_task ON task.id = user_task.task_id 
+JOIN user ON user_task.user_id = user.id 
+WHERE user.name LIKE 'Donald Duck' AND status.name = 'Not started';
+
+
+--Get all the tasks for 'Maryrose Meadows' that were created in september
+SELECT task.title, task.created, user.name 
```

Comment:
I believe the task.created doesnt really make sense, its better to be replaced by MONTH(created) as well